go/tasks/pluginmachinery/flytek8s/pod_helper.go

```diff
@@ -43,6 +43,7 @@ func ToK8sPodSpec(ctx context.Context, taskExecutionMetadata pluginsCore.TaskExe
 			Tolerations:        GetPodTolerations(taskExecutionMetadata.IsInterruptible(), c.Resources),
 			ServiceAccountName: taskExecutionMetadata.GetK8sServiceAccount(),
 			NodeSelector:       config.GetK8sPluginConfig().InterruptibleNodeSelector,
+			SchedulerName:      config.GetK8sPluginConfig().SchedulerName,
```

Comment:
I will say that we set scheduler only if it is set.

Reply:
Agree. I tested it and when the flag is unset in .yml file, the pod is using `default-scheduler`. So even with writing a value here the behavior is still matching expectations